.gitignore

```diff
@@ -65,4 +65,3 @@ timeline.xctimeline
 playground.xcworkspace
 
 buildboth.sh
-
```

Comment:
nit: this change could be removed

Reply:
This is weird though. When creating the PR I try to reduce the changes by copy paste your file, but then Github automatically remove the last empty line. My IDE is 3.5.3 with a few difference that's why I did this in browser. To be fixed.